consensus/service/src/client_api_service.rs

```diff
@@ -26,21 +25,21 @@ use std::sync::Arc;
 const PENDING_LIMIT: i64 = 500;
 
 #[derive(Clone)]
-pub struct ClientApiService<E: ConsensusEnclaveProxy, L: Ledger + Clone> {
-    enclave: E,
+pub struct ClientApiService<L: Ledger + Clone> {
```

Comment:
The changes here are small, but were tricky to get right. This struct needs to get passed to auto-generated gRPC code which runs the service, so it needs to be Clone and safe to send between threads. It's also constrained because I want the ConsensusEnclave and TxManager traits to work with automock.